apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx

```diff
@@ -119,17 +124,33 @@ export const UriAttributesSettings: FunctionComponent<AdvanceAttributeSettingsPr
                         />
                     </Form>
                     <Hint>
-                        <Trans
-                            i18nKey={
-                                "console:console:develop.features.authenticationProvider.forms.uriAttributeSettings" +
-                                ".subject.hint"
-                            }
-                        >
-                            The attribute that identifies the user at the enterprise identity provider. 
-                            When attributes are configured based on the authentication response of this IdP connection, 
-                            you can use one of them as the subject. Otherwise, the 
-                            default <Code>saml2:Subject</Code> in the SAML response is used as the subject attribute.
-                        </Trans>
+                        { isSaml ?
+                            (
+                                <Trans
+                                    i18nKey={
+                                        "console:console:develop.features.authenticationProvider.forms" +
+                                        ".uriAttributeSettings.subject.hint"
+                                    }
+                                >
+                                The attribute that identifies the user at the enterprise identity provider. 
+                                When attributes are configured based on the authentication response of this 
+                                IdP connection, you can use one of them as the subject. Otherwise, the 
+                                default <Code>saml2:Subject</Code> in the SAML response is used as the 
+                                subject attribute.
+                                </Trans>
+                            )
+                            : 
```

Comment:
Shall we follow a format like this?
```
condition
    ? statement
    : statement
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3279/commits/0211cfdea4d21643b59fe812ddcab30928cb0fe5